ARCHITECTURE.md

```diff
@@ -38,7 +38,7 @@ To set up a development machine, do the following:
 
 1. `cd` into bigcases2/docker/bigcasesbot, then launch the server by running:
 
-    `docker-compose up -d`
+    `docker compose up --build`
```

Comment:
```suggestion
    `docker compose up -d --build`
```
I think it is better to run it in detached mode and use tools like docker desktop to monitor the service logs